app/scripts/utils/path.js

```diff
@@ -1,26 +1,28 @@
-function Path (protocol, name, path) {
-  this.protocol = protocol
-  this.name = name
-  this.path = path
+export default class Path {
+  constructor (protocol, name, path) {
+    this.protocol = protocol
+    this.name = name
+    this.path = path
+  }
 
-  this.parent = function () {
+  parent () {
     var slashIdx = this.path.lastIndexOf('/')
     if (slashIdx === -1) return null
-    return new Path(protocol, name, this.path.substr(0, slashIdx))
+    return new Path(this.protocol, this.name, this.path.substr(0, slashIdx))
   }
 
-  this.append = function (fragment) {
+  append (fragment) {
     if (fragment[0] !== '/') {
       fragment = '/' + fragment
     }
-    return new Path(protocol, name, this.path + fragment)
+    return new Path(this.protocol, this.name, this.path + fragment)
   }
 
-  this.toString = function () {
+  toString () {
     return '/' + this.protocol + '/' + this.name + this.path
```

Comment:
Can we have an example of how this looks? Shouldn't there be a divider between name and path?